core/corehttp/gateway_test.go

```diff
@@ -153,6 +153,12 @@ func TestGatewayGet(t *testing.T) {
 	ns["/ipns/double.example.com"] = path.FromString("/ipns/working.example.com")
 	ns["/ipns/triple.example.com"] = path.FromString("/ipns/double.example.com")
 	ns["/ipns/broken.example.com"] = path.FromString("/ipns/" + k)
+	// We picked .man because:
+	// 1. It's a valid TLD.
+	// 2. Go treats it as the file extension for "man" files (even though
+	//    nobody actually *uses* this extension, AFAIK).
```

Comment:
This is system dependent.  http.ServeContext calls mime.TypeByExtension which in the sandbox go environment does not detect it as a `application/x-troff-man`.  See: https://play.golang.org/p/ohNcHHsVArq

Reply:
Ah. Damn, that's why `.com` wasn't working.